src/utils/traverse.js

```diff
@@ -102,6 +103,7 @@ const ORDER = {
   LogicalNotOp: ['expression'],
   LogicalOrOp: ['left', 'right'],
   MemberAccessOp: ['expression'],
+  MultiplyOp: ['left', 'right'],
```

Comment:
Thanks for the explanation. I was thrown off by the difference between the PR title and the actual changes being made. No need to submit multiple PRs, but in the future it'd be nice to at least separate distinct changes into distinct commits if not PRs.